### DISCUSSION TITLE
unchecked QSL types on Awards screens

### DISCUSSION THREAD

**Discussion by xyz667** [wavelog/wavelog#1358] [General]
Something I’m not sure if it’s a bug or on purpose:
In all the Awards screens (e.g. Awards - Germany -DOK) there is a selection „Show QSO with QSL Type“ with the different QSL types.
„QSL Card“ and „LoTW“ are always selected, the others are never selected when loading the page.
Why? Why not pre-select all of them?

**Comment by HB9HIL** [comment]
This is on purpose. Reason is that for these awards only Paper QSL and LoTW are valid QSL methods. You can't get the official Award Certificate with eQSL, QRZ, and so on.

**Comment by xyz667** [comment]
Thanks for the clarification, I don't have much experience with awards yet.
